src/main/java/com/company/presentation/controller/BusinessLogic.java

```diff
@@ -3,246 +3,207 @@
 import com.company.Util;
 import com.company.presentation.model.Board;
 import com.company.presentation.model.Boardlist;
-import com.company.presentation.model.Card;
 import com.company.presentation.view.Menu;
 import com.company.presentation.view.Menupoints;
 import com.company.service.BoardService;
 import com.company.service.BoardlistFileService;
 import com.company.service.CardFileService;
-import java.io.BufferedReader;
 import java.io.File;
 import java.io.FileInputStream;
 import java.io.FileOutputStream;
 import java.io.IOException;
-import java.io.InputStreamReader;
 import java.io.ObjectInputStream;
 import java.io.ObjectOutputStream;
 import java.util.ArrayList;
-import java.util.List;
 
 /**
- * Keeps all business logic and needs to be refactored
+ * Model Acsses occur only with Servises
  */
 public class BusinessLogic {
 
-  BoardService boardService;
-  CardFileService cardFileService;
-  Board board;
-  BoardlistFileService boardlistFileService;
-  private String input;
+  private BoardService boardService;
+  private CardFileService cardFileService;
+  private BoardlistFileService boardlistFileService;
+
   private Menu menu;
   private File file;
   private Check check;
-  private Card card;
-  private List<Board> allBoards;
-  private int board_pos;
-  private String list_pos;
+
   private String klammer_auf = "[";
   private String klammer_zu = "]";
   private String minus = "-";
 
-  public BusinessLogic() throws IOException, ClassNotFoundException {
+  private String userInputString;
+  private Print print;
+  private Input input;
+
+  public BusinessLogic() throws IOException {
     //  file = new File("SerTest.bin");
     menu = new Menu();
     check = new Check();
-    allBoards = new ArrayList();
+
     boardService = new BoardService();
     boardlistFileService = new BoardlistFileService();
     cardFileService = new CardFileService();
+    print = new Print(boardlistFileService, boardService, cardFileService);
+    input = new Input();
+
+  }
+
+  public Input getInput() {
+    return input;
+  }
+
+  public void setInput(Input input) {
+    this.input = input;
+  }
+
+  public String getUserInputString() {
+    return userInputString;
+  }
+
+  public void setUserInputString(String userInputString) {
+    this.userInputString = userInputString;
   }
 
   public void runApplication() throws IOException, ClassNotFoundException {
     Util util = new Util();
     util.createFile();
+
     if (!check.fileIsEmpty(util.getFile())) {
-      boardService.setAllBoards((List) deserializeObjeckt(util.getFile()));
+      boardService.getScreen().setAllBoards((ArrayList<Board>) deserializeObjeckt(util.getFile()));
     }
 
-    while (!String.valueOf(Menupoints.END.getAction()).equals(input)) {
+    while (!String.valueOf(Menupoints.END.getAction()).equals(userInputString)) {
 
-      input = getInput();
-      validateUserInput(input);
+      userInputString = input.getInput();
+      validateUserInput();
```

Comment:
validateUserInput is not only validating but also do main logic for determining what to show, so name is not fully describes what it does